pan_firewall/datadog_checks/pan_firewall/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2020-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-__version__ = '1.0.0'
+__version__ = '0.0.1'
```

Comment:
Why not 1.0.1?

Reply:
it's not packaged correctly, 1.0.0 isn't valid. I'm releasing a 1.0.0 after this is merged